pkg/discovery/network/openshift4.go

```diff
@@ -23,13 +23,18 @@ import (
 	"fmt"
 
 	"github.com/pkg/errors"
+	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 )
 
+const (
+	NetworkTypeCalico = "Calico"
```

Comment:
```suggestion
	CalicoNetwork = "Calico"
```
seems a better constant name

Reply:
I'm totally open to changing the constant name, but since
this constant represents the value of 'networkType' field of OCP Network.config CR in case CNI is Calico, I think it makes sense to include NetworkType in the constant name somehow.  

Maybe 'CalicoNetworkType' ?

Reply:
Is it even worth having a constant for this, let alone an exported constant? It’s only used once, and I don’t think it’s going to be used anywhere else, is it?

Reply:
Yep, 'NetworkTypeCalico' should be used only here and I don't expect it to be exported.
I think we can delete the constant definition, if in the future we'll need to support  more  networkType 2 Plugin mapping,we might consider using non exported constant values.